### PR TITLE
feat(frontend): enabledBtcTokens derived store

### PR DESCRIPTION
**Pull request by DenysKarmazynDFINITY**
# Motivation

Implement enabledBtcTokens derived store.

### CODE REVIEW

File: src/frontend/src/btc/types/btc.ts [hunk 1]
```diff
@@ -1,3 +1,6 @@
+import type { Token } from '$lib/types/token';
 import type { BitcoinNetwork as BitcoinNetworkLib } from '@dfinity/ckbtc';
 
+export type BtcToken = Token;
```
[the remark this answers is not in the document]
Comment (DenysKarmazynDFINITY) [reply src/frontend/src/btc/types/btc.ts:4]: I see your point. I don't actually need it now, can just work with `Token`. And if/when there will be a need for a separate `BtcToken`, I will create a separate PR.

File: src/frontend/src/lib/derived/tokens.derived.ts [hunk 2]
```diff
@@ -52,8 +53,13 @@ export const enabledErc20Tokens: Readable<Erc20Token[]> = derived(
 );
 
 /**
- * The following store is use as reference for the list of WalletWorkers that are started/stopped in the main token page.
+ * The following 2 stores are used as reference for the list of WalletWorkers that are started/stopped in the main token page.
```
[the remark this answers is not in the document]
Comment (DenysKarmazynDFINITY) [reply src/frontend/src/lib/derived/tokens.derived.ts]: Makes sense, duplicated it. 

Also, added a TODO comment to add unit tests when the above mentioned PR is merged.

File: src/frontend/src/btc/types/btc.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (AntonioVentilii-DFINITY) [reply src/frontend/src/btc/types/btc.ts]: sounds good

**Review by peterpeterparker — APPROVED** [review]
LGTM, thx

**Pull request merged**